web/styles/breakpoints.sass

```diff
@@ -7,3 +7,61 @@ $laptop: 1080px;
 $tablet: 900px;
 $phablet: 760px;
 $mobile: 480px;
+
+//////////////////////////
+//   SCALING CLASSES    //
+//////////////////////////
+
+.scale
+  &-row
+    &-desktop
+      @media screen and (max-width: $desktop)
```

Comment:
hm why `$desktop`? Shouldn't it be `$laptop`? 

Also this might be out of scope but why is `$desktop` `1300px`? Shouldn't `$laptop` be `1024px`, and `$2k` be `1440px` and `$4k` be `2560px`?

Reply:
I sniped the breakpoints from https://github.com/ubclaunchpad/ubclaunchpad.github.io/blob/master/css/scss/utils/_breakpoints.scss because imo they are quite reasonable

I think `$2k` etc arent very good breakpoints - very few displays render websites at native resolution (macOS for example upscales by default, and i think a lot windows is configured to do the same nowadays but maybe @chamkank can pitch in on that)

Reply:
I chose to do desktop (1300) vs laptop (1080) here because after fiddling around with screen widths for a while i found it was better to break the homepage on this

Reply:
tl;dr: I will fiddle around with changing the breakpoints, but imo `desktop laptop tablet phablet mobile` are good points to break at because those are the ones where things can **really look bad**, and we should focus on making accommodating those, since if it looks good at `1920` chances are itll look okay at most reasonable screen sizes past that and we dont need to make special scaling classes for them

Reply:
actually no imma leave `desktop` as is - i think `1300` is a good point to define "alright, at this width, we switch into desktop layout", `1920` is way to big to be useful for breakpoints

Reply:
Yeah, I agree that we don't need breakpoints past the standard non-mobile resolution (2k and 4k are probably excessive).

I'm not sure why we need both `desktop` and `laptop`... don't most displays look the same between a desktop and laptop? and i grabbed the numbers i came up with for $2k etc from chrome responsive mode, but i don't understand where they got those pixels from so scratch that. But I think we should keep it consistent with the standard resolution like `1080p (1920 x 1080)`.

so I'd say we just have one break point, 1920px, for non-mobile resolution and keep the lower ones the same?

Reply:
The problem with using `1920` as a breakpoint is that I dont really see a case where something looks good from like `1000` to `1920` and then abruptly looks awful after

I think you might be looking at the breakpoints a bit differently - these are intended to be the "line" between "classes" of display types, where each class of display type might need some kind of CSS adjustment. `laptop` here is useful as a point to use where a slightly smaller screen is juuuusssst to small to comfortably display something... in this case, the sun at the top banner. A `1920` is not useful here, because theres not much that will change *past* `1920` where CSS adjustments need to be made, whereas its too wide to say this is the point where something becomes a desktop... I think `1300`, from what ive noticed fiddling with this PR, is a comfortable entry into the `desktop layout` zone

Reply:
oh i see, i understand now. Yeah I was thinking about it differently, but that makes sense.